docs(controller): document audit logging (#7269)
influxdata/docs-v2: content/telegraf/controller/audit-logs/enable-configure.md

1. Sign in as an **Owner** or **Administrator**.
2. Navigate to the **Settings** page from the left navigation menu.
3. In the **Audit Logging** section, select a value from **Audit log retention**.
4. Click **Save**.

<!-- TODO: screenshot of the Settings > Audit Logging section with the Audit log retention dropdown expanded, showing the 30 days / 3 months / 6 months / 1 year / 2 years / Infinite options. Save to /static/img/telegraf/controller-settings-audit-retention.png and replace this comment with: {{< img-hd src="/img/telegraf/controller-settings-audit-retention.png" alt="Telegraf Controller audit log retention dropdown" />}} -->

The new retention value takes effect immediately.
The next cleanup run removes any entries that fall outside the new window.

### Set the initial retention at startup

Use the `AUDIT_LOG_RETENTION` environment variable to seed the retention
period when {{% product-name %}} initializes its settings on first startup.

```bash
export AUDIT_LOG_RETENTION=8760
```

> [!Note]
> `AUDIT_LOG_RETENTION` only sets the initial value.
> After first startup, the database is authoritative --- update retention

> > from the **Settings** page.

Reviewer: ```suggestion
> #### Update retention in Settings
> `AUDIT_LOG_RETENTION` only sets the initial value.
> After first startup, the database is authoritative.
> To update audit log retention, use the **Settings** page.
```

## Forward audit events

{{% product-name %}} can forward each audit event to one or more external
destinations in addition to writing it to local storage.
Forwarders are configured **at startup only** and run independently --- you

> can enable any combination of syslog, webhook, and file forwarders.

Reviewer: ```suggestion
Forwarders are configured **at startup only** and run independently, which lets you enable any combination of syslog, webhook, and file forwarders.
```

### Forward to a webhook

Forward audit events as JSON `POST` requests to an HTTP webhook.

```bash
export AUDIT_WEBHOOK_URL=https://siem.example.com/ingest
export AUDIT_WEBHOOK_AUTH_HEADER="Bearer xxxxxxxxxxxx"
```

| Variable                    | Description                                            | Required |
| :-------------------------- | :----------------------------------------------------- | :------- |
| `AUDIT_WEBHOOK_URL`         | Full URL the webhook receives `POST` requests at       | Yes      |
| `AUDIT_WEBHOOK_AUTH_HEADER` | Optional value sent in the `Authorization` HTTP header | No       |

The webhook forwarder retries each event up to three times with backoff and a
10-second request timeout.
Events that return `408`, `429`, or `5xx` responses are retried; events that
return other `4xx` responses are dropped.
{{% product-name %}} honors a `Retry-After` response header when present.

### Append to a file

Append each event to a file as a single JSON object per line (`.jsonl`).

```bash
export AUDIT_FILE_PATH=/var/log/telegraf-controller/audit.jsonl
```

The path must be writable by the {{% product-name %}} process.

> {{% product-name %}} does not rotate or trim this file --- pair it with a

Reviewer: ```suggestion
{{% product-name %}} does not rotate or trim this file--pair it with a
```
